x-pack/solutions/security/plugins/security_solution/public/siem_migrations/dashboards/components/migration_status_panels/migration_result_panel.tsx

```diff
@@ -0,0 +1,308 @@
+              <>
+                <MigrationsLastError
+                  message={migrationStats.last_execution.error}
+                  migrationType="dashboard"
+                />
+                <EuiSpacer size="m" />
+              </>
+            )}
+            <EuiFlexGroup direction="column" alignItems="stretch" gutterSize="m">
+              <EuiFlexItem grow={false}>
+                <EuiFlexGroup direction="row" alignItems="center" gutterSize="s">
+                  <EuiFlexItem grow={false}>
+                    <EuiIcon type={AssistantIcon} size="m" />
+                  </EuiFlexItem>
+                  <EuiFlexItem>
+                    <PanelText size="s" semiBold>
+                      <p>{i18n.DASHBOARD_MIGRATION_SUMMARY_TITLE}</p>
+                    </PanelText>
+                  </EuiFlexItem>
+                </EuiFlexGroup>
+              </EuiFlexItem>
+              <EuiFlexItem>
+                <EuiPanel hasShadow={false} hasBorder paddingSize="m">
+                  <EuiFlexGroup direction="column" alignItems="stretch" justifyContent="center">
+                    <EuiFlexItem>
+                      {isLoadingTranslationStats ? (
+                        <CenteredLoadingSpinner />
+                      ) : (
+                        translationStats && (
+                          <>
+                            <EuiText size="m" css={{ textAlign: 'center' }}>
+                              <b>{i18n.DASHBOARD_MIGRATION_SUMMARY_CHART_TITLE}</b>
+                            </EuiText>
+                            <TranslationResultsChart translationStats={translationStats} />
+                            <TranslationResultsTable translationStats={translationStats} />
+                          </>
+                        )
+                      )}
+                    </EuiFlexItem>
+                    <EuiFlexItem grow={false}>
+                      <EuiFlexGroup direction="column" alignItems="center">
+                        <EuiFlexItem>
+                          <SecuritySolutionLinkButton
+                            deepLinkId={SecurityPageName.siemMigrationsDashboards}
+                            path={migrationStats.id}
+                          >
+                            {i18n.DASHBOARD_MIGRATION_VIEW_TRANSLATED_DASHBOARDS_BUTTON}
+                          </SecuritySolutionLinkButton>
+                        </EuiFlexItem>
+                      </EuiFlexGroup>
+                    </EuiFlexItem>
+                  </EuiFlexGroup>
+                </EuiPanel>
+              </EuiFlexItem>
+            </EuiFlexGroup>
+            <DashboardMigrationsUploadMissingPanel
+              migrationStats={migrationStats}
+              topSpacerSize="s"
+            />
+          </EuiPanel>
+        </EuiAccordion>
+      </EuiPanel>
+    );
+  }
+);
+DashboardMigrationResultPanel.displayName = 'MigrationResultPanel';
+
+const TranslationResultsChart = React.memo<{
```

Comment:
Translation results related components - `TranslationResultsChart` and `TranslationResultsTable`, look like could be common? Does not need to be done tin this PR, but maybe something we could improve in one of the followups.

Reply:
Yeah.. I thought about it and i thought it was not worth the type gymnastics for now so i kept it separate. Probably later.